app/views/shared/_footer.html.erb

```diff
@@ -5,18 +5,16 @@
         <div class="grid-s-12 grid-m-6">
           <div class="newsletter">
             <h2 class="title -big">Newsletter</h2>
-            <p class="text -leading">To receive insight2impact updates, <button type="button" class="text -highlight -button js-subscribe">subscribe</button> to our newsletter.</p>
+            <p class="text -leading">
+              Sign up <button type="button" class="text -highlight -button js-subscribe">here</button> for our monthly Data Briefing on data and innovation in the financial services sector. 
+            </p>
           </div>
         </div>
 
         <div class="grid-s-12 grid-m-6">
           <div class="contact-us">
             <h2 class="title -big">Contact us</h2>
-            <p class="text -leading">Can’t find what you are looking for? Know about a financial dataset that’s not currently
-              available on our data portal thar we should add?</p>
-
-            <p class="text">Please write to us and give us your feedback
-              <a class="text -highlight -twitterlink" href="https://twitter.com/i2ifacility" rel="noopener noreferrer" target="_blank">@i2ifacility</a></p>
+            <a class="text -highlight -leading" href="https://twitter.com/i2ifacility" href="mailto:info@i2ifacility.org">info@i2ifacility.org</a>
```

Comment:
There's no need of the first `href`, we want the link to open the default email app of the user with our  email direction. That's enough.

Reply:
👍